scripts/index.js

```diff
@@ -0,0 +1,157 @@
+// default variable for temperature--used as parameter in several functions
+let currentTemp = 67;
+let isCelsius = false;
+
+// helper function to be used when increasing/decreasing temperature
+// currentTemp "shadows" currentTemp global variable
+const updateTemperature = currentTemp => {
+    const tempContainer = document.getElementById("temp-integer");
+    tempContainer.textContent = currentTemp;
+    changeTempColor(currentTemp);
+    changeLandscape(currentTemp);
+};
+
+// converts temp to celsius or fahrenheit
+// Math.floor creates a slight issue where temp continues to decrease when converted
+const convertTemp = () => {
+    const tempConvertBtn = document.getElementById("temp-converter-button");
+    if (isCelsius === true) {
+        currentTemp = Math.floor((currentTemp * 9/5) + 32);
+        isCelsius = false;
+        tempConvertBtn.textContent = "Celsius"
+        updateTemperature(currentTemp);
+    }
+    else if (isCelsius === false) {
+        currentTemp = Math.floor((currentTemp - 32) * 5/9);
+        isCelsius = true;
+        tempConvertBtn.textContent = "Fahrenheit"
+        updateTemperature(currentTemp);
+    }
+};
+
+// increases temperature by 1
+const increaseTemperature = () => {
+    currentTemp += 1;
+    updateTemperature(currentTemp);
+};
+
+// decreases temperature by 1
+const decreaseTemperature = () => {
+    currentTemp -= 1;
+    updateTemperature(currentTemp);
+};
```

Comment:
Just note that you can combine these functions using a paramter.

```suggestion
// increases temperature by 1
const changeTemperature = (diffTemp) => {
    currentTemp += diffTemp;
    updateTemperature(currentTemp);
};
```

Then you can set the event handler with

```javascript
tempIncreaseBtn.addEventListener("click", () => changeTemperature(-1) );
```